Syntaxes/Chapel.tmLanguage

```diff
@@ -269,8 +269,7 @@
 		<dict>
 			<key>match</key>
 			<string>(?x)\b(
-              	abs | max | min | sqrt | write | writeln | read | readln | open | close | 
-				exit
+              	 abs | close | exit| max | min | open | read | readln | sqrt | write | writeln
```

Comment:
It looks like there is a missing space here after exit (though I wouldn't feel obligated to alphabetize any line you didn't have to touch)